spec/labor/markdown_parser_spec.rb

```diff
@@ -81,12 +81,27 @@ def generate_and_parse_markdown(raw_markdown)
       expect(result).to include "<a", "<em"
     end
 
+    it "works in ul/li tag" do
+      mention = <<~DOC
+        `@#{user.username}` one two, @#{user.username} three four:
+          - `@#{user.username}`
+      DOC
+      result = generate_and_parse_markdown(mention)
+      expect(result).to eq("<p><code>@#{user.username}</code> one two, <a class=\"comment-mentioned-user\" href=\"http://localhost:3000/#{user.username}\">@#{user.username}</a>\n three four:</p>\n\n<ul>\n<li><code>@#{user.username}</code></li>\n</ul>\n\n")
```

Comment:
would it possible over here not to have `localhost:3000` hard checked?

Reply:
Good catch! I'll go ahead and change that.